README.md

```diff
@@ -1 +1,170 @@
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Template</summary>
+  <ul>
+    <li><a href="https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=2tXla7pFfIxmrpdc-0/">Figma</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Key Feature 1]**    Error correction by Microverse Linter configuration
+- **[Key Feature 2]**    Responsive design to support running in any device
+- **[Key Feature 3 ]**   Documented in a professional way 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy and running the project consider below items:
+
+### Prerequisites
+
+In order to run this project you need:
+
+To get a local copy of my project you can use below link.
+
+"git@github.com:Jawad-Nazari/Portfolio-Project-Mobile-first.git"
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Choose a directory
+- Open a Terminal
+- write: git clone "git@github.com:Jawad-Nazari/Portfolio-Project-Mobile-first.git"
+
+### Install
+
+   In order to run or modify this project you need to install the following:
+   - git
+   - code editor ( VS Code Suggested )
+   - browser  (Chrome Prefered)
+
+### Usage
+
+This project will be used to display my portfolios:
+
+
+### Run tests
+
+To run and test, open the below HTML file in your browser:
+
+- index.html
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jawad Nazari**
+
+- GitHub: [@Jawad-Nazari](https://github.com/Jawad-Nazari)
+- LinkedIn: [Jawad-Nazari](https://www.linkedin.com/in/Jawad-)
+- Twitter: [@nazari_jawad58](https://twitter.com/nazari_jawad58)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [1] **[Portfolio pages will be added]**
+- [2] **[Contact details will be added]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I'd like to thank Microverse for providing this chance to enrich my experience.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] You did a great job 😃 at adding the MIT LICENCE, but I see that when clicking on the link to open the page of LICENCE I see that it is redirecting me to the page not found, so please 😊 try to put the correct link for the MIT LICENCE to let anyone when clicking on the link of LICENCE redirect him/her to the page of the LICENCE 💯